src/net/packets/outbound-packets/sidebar-widget-packet.ts

```diff
@@ -0,0 +1,22 @@
+import { ByteBuffer } from '@runejs/common';
+import { OutboundPacket } from '../packets';
+
+interface SidebarWidgetData {
+    widgetId: number;
+    sidebarId: number;
+}
+
+export const sidebarWidgetPacket: OutboundPacket<SidebarWidgetData> = {
```

Comment:
Should we move the endcoders into an encoders into an encoders folder?

Reply:
I wasn't sure if they'd get large enough, but I suppose we can if y'all wanna

Reply:
I think it would look cleaner as we add more.